cli_tools_tests/e2e/gce_ovf_import/test_suites/ovf_instance_import/ovf_instance_import_test_suite.go

```diff
@@ -125,10 +128,53 @@ func TestSuite(
 	testsMap[e2e.Wrapper][instanceImportDefaultServiceAccountNoAccessScopeTestCase] = runInstanceImportDefaultServiceAccountNoAccessScope
 	testsMap[e2e.Wrapper][instanceImportNoServiceAccountTestCase] = runInstanceImportNoServiceAccount
 
+	// wrapper only tests
+	testsMap[e2e.Wrapper][junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v] %v", e2e.Wrapper, "Allow import to work when SSD quota is exhausted"))] = fallbackWhenSSDQuotaExhausted
+
 	e2e.CLITestSuite(ctx, tswg, testSuites, logger, testSuiteRegex, testCaseRegex,
 		testProjectConfig, testSuiteName, testsMap)
 }
 
+func fallbackWhenSSDQuotaExhausted(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+	testProjectConfig *testconfig.Project, testType e2e.CLITestType) {
+	suffix := path.RandString(5)
+	zone := "us-central2-a"
```

Comment:
Did you setup this zone's quota to make it happen?

Reply:
The quota for that zone was already limited to 500GB. Since the test depends on having insufficient quota, I added the assertion that creating a 600GB disk fails. (The OVF has a 1000GB disk).